### DISCUSSION TITLE
Will Istio 1.23 support K8s 1.31 or not?

### DISCUSSION THREAD

**Discussion by westwind-sc** [istio/istio#52144] [General]
According to the Release Plan, 
Istio `1.23`, Expected Release Date: Expected `August 12th, 2024` https://github.com/istio/istio/wiki/Istio-Release-1.23
K8s `1.31.0`, will Release at  `August 13th, 2024`, https://github.com/kubernetes/sig-release/tree/master/releases/release-1.31

Will Istio 1.23 plan to support K8s 1.31 or not?

**Comment by westwind-sc** [comment]
K8s 1.31.0 just released https://github.com/kubernetes/kubernetes/releases/tag/v1.31.0
As Istio 1.23.0 still in RC1, could we consider make Istio 1.23 support K8s 1.31?
With discussion in https://github.com/istio/istio/discussions/45315, seems need acceptance tests.

loop Istio 1.23 Release Managers for awareness.  @thedebugger @hzxuzhonghu  @mikemorris

**Reply by craigbox** [reply]
The plan is for 1.23.1 to support 1.31.
